packages/std/src/ibc.rs

```diff
@@ -35,6 +35,9 @@ pub enum IbcMsg {
         amount: Coin,
         /// when packet times out, measured on remote chain
         timeout: IbcTimeout,
+        /// optional memo
+        #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Seems like schemars is not aware of this annotation. It might be easier to remove it. In `WasmMsg::Instantiate` we already have an example where `Option<String>` in Rust becomes a non-optional string field in Go, so that should work.

Reply:
Is the annotation even relevant for schemars? I don't think it would influence the schema. But I agree, it should also work without it.

Reply:
Oh, maybe not. `memo` is not in the required fields list, making it omitable. And `null` is valid still.

Anyhow, let's stick to well known flows here